ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_volume_collection.py

```diff
@@ -0,0 +1,720 @@
+    - inval
+    - exchange
+    - exchange_dag
+    - hyperv
+    - sql2005
+    - sql2008
+    - sql2012
+    - sql2014
+    - sql2016
+    - sql2017
+    type: str
+    description:
+    - Application ID running on the server.
+  app_server:
+    required: False
+    type: str
+    description:
+    - Application server hostname.
+  app_service:
+    required: False
+    type: str
+    description:
+    - If the application is running within a windows cluster environment then this is the instance name of the service running within the cluster environment.
+  app_sync:
+    choices:
+    - none
+    - vss
+    - vmware
+    - generic
+    required: False
+    type: str
+    default: none
+    description:
+    - Application synchronization.
+  demote:
+    required: False
+    type: bool
+    description:
+    - Release ownership of the specified volume collection. The volumes associated with the volume collection will set to offline and
+      a snapshot will be created, then full control over the volume collection will be transferred to the new owner. This option can be used
+      following a promote to revert the volume collection back to its prior configured state. This operation does not alter the configuration on
+      the new owner itself, but does require the new owner to be running in order to obtain its identity information. This operation is not supported
+      for synchronous replication volume collections.
+  description:
+    required: False
+    type: str
+    description:
+    - Text description of volume collection.
+  handover:
+    required: False
+    type: bool
+    description:
+    - Gracefully transfer ownership of the specified volume collection.This action can be used to pass control of the volume collection to the downstream
+      replication partner.Ownership and full control over the volume collection will be given to the downstream replication partner. The volumes associated
+      with the volume collection will be set to offline prior to the final snapshot being taken and replicated, thus ensuring full data synchronization
+      as part of the transfer. By default, the new owner will automatically begin replicating the volume collection back to this node when the handover completes.
+  invoke_on_upstream_partner:
+    required: False
+    type: bool
+    description:
+    - Invoke handover request on upstream partner. This operation is not supported for synchronous replication volume vollections.
+  is_standalone_volcoll:
+    required: False
+    type: bool
+    default: False
+    description:
+    - Indicates whether this is a standalone volume collection.
+  metadata:
+    required: False
+    type: dict
+    description:
+    - User defined key-value pairs that augment an volume collection attributes. List of key-value pairs. Keys must be unique and non-empty.
+    - When creating an object, values must be non-empty. When updating an object, an empty value causes the corresponding key to be removed.
```

Comment:
Why does this new line start with a "-" while other description that span multiple lines do not?

Reply:
removed